digid_eherkenning/models/base.py

```diff
@@ -240,6 +229,34 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     def clean(self):
-        if not self.certificate:
-            raise ValidationError(_("You must select a certificate"))
         super().clean()
+
+        # require that a certificate is configured
+        if not ConfigCertificate.objects.for_config(self).exists():
+            raise ValidationError(
+                _(
+                    "You must prepare at least one certificate for the {verbose_name}."
+                ).format(verbose_name=self._meta.verbose_name)
```

Comment:
it can, but I don't want to tie it to that even though it's currently the same implementation. I suspect you make this remark because `_meta` seems private? If that's the case, it's not private, it's publicly documented and explicitly considered public API in the Django documentation.

In this case, I'm really referring to the model as a whole rather than a single instance, the lines between that are blurry with django-solo :grimacing:

Reply:
No it was mostly because I saw that it was the same implementation!